roles/ceph-osd/tasks/scenarios/lvm.yml

```diff
@@ -1,12 +1,81 @@
 ---
+
+- name: list all lvm osds
+  command: ceph-volume lvm list
+  register: ceph_volume_lvm_list
+  failed_when: False
+  changed_when: False
+  tags:
```

Comment:
I think you meant "check_mode: no" here 

Reply:
fixed